mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraAttachmentDAOV2.java

```diff
@@ -142,7 +142,7 @@ public CassandraAttachmentDAOV2(BlobId.Factory blobIdFactory, Session session) {
     private PreparedStatement prepareDelete(Session session) {
         return session.prepare(
             QueryBuilder.delete().from(TABLE_NAME)
-                .where(eq(ID, bindMarker(ID))));
+                .where(eq(ID_AS_UUID, bindMarker(ID_AS_UUID))));
```

Comment:
it looks like a fixup, shoud it be squashed with the previous commit?

Reply:
Please put commit link, I don't have it in github PR